scheduler/scheduler.go

```diff
@@ -30,7 +30,7 @@ import (
 	"path"
 )
 
-type leaderFunc func(<-chan interface{}, coordclient.Connection, dao.ControlPlane, string, int)
+type leaderFunc func(<-chan interface{}, coordclient.Connection, dao.ControlPlane, *facade.Facade, string, int)
```

Comment:
We could have used the dao, but, if you remember, you said that was a terrible idea, which is why I added the GetRunningServicesForHosts facade method.